pina/solvers/solver.py

```diff
@@ -9,76 +9,51 @@
 import sys
 
 
-class MultipleSolversInterface(lightning.pytorch.LightningModule, metaclass=ABCMeta):
+class _PINASolverInterface(lightning.pytorch.LightningModule, metaclass=ABCMeta):
     """
-    Solver base class. This class inherits is a wrapper of
-    LightningModule class, inheriting all the
-    LightningModule methods.
+    SolverInterface base class. This class is a wrapper of the LightningModule
     """
 
     def __init__(self,
-                 models,
                  problem,
-                 optimizers,
-                 schedulers,
-                 extra_features=None,
-                 use_lt=True):
+                 use_lt,
+                 extra_features):
```

Comment:
I would remove extra_features here, I think it's better to insert feature directly in the `model` to simplify the solver structure

Reply:
Sure!